corehq/apps/style/static/style/less/legacy/app_manager.less

```diff
@@ -3,6 +3,13 @@
   top: 1.5em;
 }
 
+.app-comment {
+    font-size: 12pt;
+    font-style: italic;
```

Comment:
I'd drop this - these comments may get long, and blocks of italic text are tough to read. http://accessibility.psu.edu/legibility/textblocks/